dvc/cache/local.py

```diff
@@ -114,6 +114,7 @@ def status(
         jobs=None,
         show_checksums=False,
         download=False,
+        log_missing=True,
```

Comment:
We've been thinking about revisit logging/info interface for a long time, since passing a chain of such flags is not the prettiest thing ever :slightly_frowning_face: But there is no simple better way for now, so this will work for now :+1: